File: Sakila_Net/Sakila_Net/VB_SakilaLib/Neg/NegActor.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegActor
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOActor.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( actor as ClActor) as Boolean
                        Return DAO.DAOActor.SelectOne(actor)     
                    End Function
                    Public  Shared Function crear( actor as ClActor) as Integer
                        Return DAO.DAOActor.SqlInsert(actor)     
                    End Function
                    Public  Shared Function eliminar( actor as ClActor) as Integer
                        Return DAO.DAOActor.SqlDelete(actor)     
                    End Function
                    Public  Shared Function actualizar( actor as ClActor) as Integer
                        Return DAO.DAOActor.SqlUpdate(actor)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegActor.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegActor
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOActor.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClActor actor) 
        {
            return DAO.DAOActor.sqlSelect(actor);
        }
        public  static int crear(ClActor actor)
        {
            return DAO.DAOActor.sqlInsert(actor);
        }
        public  static int eliminar(ClActor actor)
        {
            return DAO.DAOActor.sqlDelete(actor);
        }
        public  static int actualizar(ClActor actor)
        {
            return DAO.DAOActor.sqlUpdate(actor);
        }
    }
}
